modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/RoleResourceTest.java

```diff
@@ -278,6 +292,21 @@ protected Role testDeleteOrganizationRoleUserAccountAssociation_addRole()
 		return _user.getUserId();
 	}
 
+	@Override
+	protected Role
+			testDeleteRoleByExternalReferenceCodeUserAccountAssociation_addRole()
+		throws Exception {
+
+		return _getRoleWithExternalReferenceCode();
```

Comment:
why are you not calling the _addRole method like in `testDeleteRoleUserAccountAssociation_addRole` ?

Reply:
Please see my last comment below.

Reply:
Please see my last comment below.